README.md

```diff
@@ -7,6 +7,55 @@
 These are the officially supported images.</br>
 For the downstream / experimental images, check the [downstream](https://github.com/BredOS/images/tree/downstream) branch.</br>
 
+## Modifying the images to add new device
+We may want to add support for a new device or add support for a temperature or touch screen sensor on the i2c bus. This is different from the x86 world where the kernel is able to detect the hardware automatically with ACPI, etc. In the ARM world, the kernel needs to be told explicitly about the hardware. 
+
+Currently, there are 2 ways to support a new device:
```

Comment:
Referring to hardware modules as devices can confuse the reader, making him think of whole boards.
```suggestion
Currently, there are 2 ways to alter the device tree of a system:
```